_guidelines-v4/05-mensural.md

```diff
@@ -10,8 +10,8 @@ sectionid: "mensural"
 
 This chapter describes the module for encoding mensural notation from the late 13th century to about 1600. Historically, mensural notation preceded the development of Common Music Notation (CMN) and it included a wide range of features that persist in CMN and that can be encoded in a standard manner in MEI. In mensural notation, pitches are notated as in CMN, leaving out here the major exception of *musica ficta*. The pitch is given by the position of the note on the staff and the current clef as in CMN, and the mensural module introduces no modification to MEI regarding how pitches are encoded.
 
-There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests for appropriately encoding mensural durations. One of the main particularities is that the actual duration of a note is not given only by its symbol but also by position and the context in which the symbol appears. The general context is given by one of the 16 mensural *species* provide four levels of division: *modus major*, *modus minor*, *tempus* and *prolatio*. Depending on the context, certain rules must be applied in order to determine the duration of a note. In these cases, encoding both the sign and its actual duration is highy desirable.
+There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests in order to appropriately encode mensural durations. One of the main features is that the actual duration of a note is specified not only by its symbol but also by its position and the context in which the symbol appears. The general context is given by one of the 16 mensural *species* provided by the four levels of division: *modus major*, *modus minor*, *tempus*, and *prolatio*. Depending on the context, certain rules must be applied to determine the duration of a note. Therefore, encoding information on both the sign and its actual duration is highly desirable.
```

Comment:
`...is specified not only by its symbol...` I would probably change that to something like `...is specified by both its visual appearance, its position relative to other notes, and the mensuration species, indicated by a mensuration symbol on the staff.` It can then go on to describe the mensuration species, etc.

This removes the 'not onlyism' (makes the declaration more direct and does not cause the reader to invert the meaning), and clarifies, up front, what is meant by 'context', which can otherwise be a vague term.

Reply:
Hi @ahankinson! Thanks for checking this out.

Ok, so, I am modifying it to clarify upfront what was meant by 'context'. You are right, it was kind of confusing. Here "context" is meant to stand for "mensuration" and sometimes the term is used to stand for preceding and following notes. So I changed that.

I would like to keep the "only-ism", though. Because this is an issue that distinguishes mensural notation from CMN. You cannot just use the note symbol to know the relative duration. So I am rephrasing it a bit. This is how the text looks like now:

>There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests in order to appropriately encode mensural durations. In contrast to common (Western) music notation, the actual duration of a note in mensural notation is specified not only by its visual appearance (i.e., the note shape), but also by its position relative to other notes and the mensuration species (equivalent to the modern concept of meter). The 16 mensural *species* are provided by the four levels of division: *modus major*, *modus minor*, *tempus*, and *prolatio*; each of which can have a binary or ternary value. For triple mensurations, certain rules must be applied to determine the duration of a note based on the context (i.e., its position relative to other notes). Therefore, encoding information on both the sign and its actual duration is highly desirable.

Do you have any issues with it? (I am asking before I push my changes.)